src/components/views/rooms/EventTile.js

```diff
@@ -537,6 +538,7 @@ module.exports = withMatrixClient(React.createClass({
             mx_EventTile_bad: isEncryptionFailure,
             mx_EventTile_emote: msgtype === 'm.emote',
             mx_EventTile_redacted: isRedacted,
+            mx_EventTile_sender: isSender,
```

Comment:
The name doesn't fully convey what this class means. How about `mx_EventTile_ownEvent`?